### PR TITLE
Draft for annoucement on mailing list

### LINKED ISSUE #44: Write on mailing list to showcase and get feedback
The next step is to write a mail to the HTTP WG mailing list (https://lists.w3.org/Archives/Public/ietf-http-wg/). Some content I would like to include:

- Why are resumable uploads useful (similar to the abstract)?
- Explain that this work is related to the tus project.
- Mention that the tus project has seen adoption, so people are interested in standardized upload protocols and ready-to-use software.
- Mention that many services also have proprietary approach (maybe?)
- Mention that tus has been discussed on the mailing list before (maybe?)

### PR DESCRIPTION
**Pull request by Acconut**
This is a brief draft to announce our work on tus v2 to the HTTP WG mailing list (https://lists.w3.org/Archives/Public/ietf-http-wg/). It briefly explains what resumable uploads are, points to our draft, and provides some background on tus to show that similar systems have already been widely adopted.

Please let me know if you have any feedback regarding it:
- Any other points that should be mentioned?
- Should anything be stressed or emphasized more?
- Should anything be left out?
- Is the style of the writing appropriate?

Fixes https://github.com/tus/tus-v2/issues/44

### CODE REVIEW

**Review by guoye-zhang — APPROVED** [review]
Looks good! I would mention that there is a difference between this draft and the current version of the protocol in tus.io. Maybe say this is a new version of the tus protocol?

File: etc/mail-annoucement.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Hello HTTP working group,
+
+in the past months we have been working on a draft for resumable file uploads using HTTP and want to present our work to you. To briefly recap, resumable uploads allow resuming an upload where it has been stopped. For example, after the connection has been interrupted. A 100 MB upload, which was interrupted after transmitting 70 MB, only needs to transmit the remaining 30 MB.
+
+There is support in HTTP for resumable downloads using the Range header, but no support for resumable uploads so far. There have been attempts to standardize resumable uploads, such as [1], but none completed this journey yet to our knowledge.
+
+We wanted to address this shortage and came up with a draft [2] (the GitHub repository is at [3]). We are interested any feedback on this and see if it is possible to get resumable uploads adopted.
```
Comment (smatsson) [inline etc/mail-annoucement.txt]: I think it's missing an "in": "We are interested IN any feedback on this..."

File: etc/mail-annoucement.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+In case you are interested in the background of this draft: The origin is within the tus project [4], which has been developing a HTTP-based protocol for resumable uploads [5] since 2013 (tus was also posted on this mailing list at the time [6]). Furthermore, we also provide various open-source implementations [7] to allow easy usage on the web, in mobile applications, desktop application, or server environments. tus has seen great adaption, proving that there is a demand for an open-source solution providing resumable uploads.
+
+We hope to bring resumable uploads to more people. For this, adopting resumable uploads into HTTP would be a great step. There is also interest in including support for resumable uploads natively into platforms, like browsers and mobile SDKs, so that developers do not have to bring their own library for resumable uploads.
+
+So we have taken the main uploading process from our tus protocol and reworked it into a self-containing draft, which we want to present to you!
```
Comment (smatsson) [inline etc/mail-annoucement.txt]: Minor: The word "so" might not be needed here. Seems redundant.

**Review by smatsson — COMMENTED** [review]
Very nice @Acconut! :) To me we seem to tick all the boxes. Can't really answer for the last of your questions though but from other messages to the mailing list I've seen it seems appropriate. Maybe @LPardue have more insights?

File: etc/mail-annoucement.txt [hunk 3]
```diff
@@ -0,0 +1,27 @@
+Hello HTTP working group,
+
+in the past months we have been working on a draft for resumable file uploads using HTTP and want to present our work to you. To briefly recap, resumable uploads allow resuming an upload where it has been stopped. For example, after the connection has been interrupted. A 100 MB upload, which was interrupted after transmitting 70 MB, only needs to transmit the remaining 30 MB.
+
+There is support in HTTP for resumable downloads using the Range header, but no support for resumable uploads so far. There have been attempts to standardize resumable uploads, such as [1], but none completed this journey yet to our knowledge.
```
Comment (LPardue) [inline etc/mail-annoucement.txt]: I would reorder these a bit to say something along the lines of 

"We are all familiar with connectivity disruptions affecting our Internet activities. One example, is when a large file download is interrupted; say a 100 MB file download encounters a network loss after the client receives 70 MB. Fortunately, resumable HTTP downloads using range requests are a widely deployed standard feature that allows clients to fetch the remaining 30 MB only, saving time and resources for both endpoints. However, in the opposite direction, there is not a standard convention for resuming HTTP uploads. 

Across the HTTP ecosystem there are several different approaches to providing resumable uploads. We are aware of at least one attempt to try and standardize an approach [1], but to our knowledge none have succeeded in being adopted and driven to conclusion.

File: etc/mail-annoucement.txt (hunk 1 above)
Comment (LPardue) [reply etc/mail-annoucement.txt]: Following on from my above suggestion, I would tweak this para to say

"We believe resumable uploads are a common problem and that there is value in a standard resumable upload approach. We've been working on a document [2] [3] that uses HTTP to solve what we believe to be the core problem set, while also allowing for extended use cases. We are bringing this to the list to understand if there is interest in the working group to solve the problem, and whether our document is a good basis for a solution.

**Comment by Acconut** [comment]
Thank you all very much for your feedback!

**Comment by Acconut** [comment]
I sent out the mail to the list and got an automated response that it will be reviewed manually as it is my first entry in the mailing list. I will keep you updated.

Going to close this PR, as the matter is done.

**Pull request closed without merging**